config/initializers/devise.rb

```diff
@@ -12,6 +12,8 @@
   # available as additional gems.
   require 'devise/orm/active_record'
 
+  config.authentication_keys = [ PerDistrict.new.educator_login_field ]
```

Comment:
This is the change most likely to impact other parts of the system. 

Testing locally, I verified that:
1. If `config.authentication_keys` is set to `[:email, :login_name]`, authentication breaks for all districts.
2. This strategy allows authentication to work for both types of login.